app/workers/user_cleaner_job.rb

```diff
@@ -15,7 +15,7 @@ def perform
     # Note that Rails.env.production? is not sufficient in this context
     # as both mampf-experimental and mampf-next also run in production mode.
     production_name = ENV.fetch("PRODUCTION_NAME", nil)
-    return unless production_name == "mampf"
+    return if production_name != "mampf" && production_name != "mampf-vignettes"
```

Comment:
For vignettes, the user cleaner should be disabled. Otherwise, study data may be destroyed as a result.

Reply:
Note the `unless` keyword.

Reply:
Oh fail, my bad, you're absolutely right. I will just revert this change I guess.